packages/tlschema/src/misc/TLHandle.ts

```diff
@@ -25,6 +25,7 @@ export interface TLHandle {
 	label?: string
 	type: TLHandleType
 	canSnap?: boolean
+	canIndependentSnap?: boolean
```

Comment:
I wonder whether it makes sense to replace these two booleans with one `snapType` enum to make the api a bit less messy.

i.e. we could deprecate `canSnap`, then add a `snapType?: 'point' | 'align' | `none`

? 

then we could log an error if they supply both `canSnap` and `snapType`.